lib/schemas/HAR/postData.json

```diff
@@ -1,7 +1,7 @@
 {
   "$id": "postData.json#",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "type": "object",
+  "type": ["object", "null"],
   "optional": true,
   "required": [
```

Comment:
There is `text` field which is not mentioned as required but it also does not contain type `null`. Is it intended?

Reply:
Thanks, will update this. `text` is used but also optional

Reply:
Got it, the shouldn't it be described under `required` array?

Reply:
I will add a `null` option here as well, on looking at the har schemas defined [here](https://github.com/ahmadnassri/har-schema/blob/master/lib/postData.json) and [here](http://www.softwareishard.com/har/viewer/) text seems optional as well, unlike the description